ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/theme/DarkTheme.java

```diff
@@ -1328,6 +1328,11 @@ public String checkBoxIconColor() {
   }
 
   @Override
+  public String checkBoxIndeterminateIconColor() {
```

Comment:
getDisabledMenuColor() return the same color

Reply:
fixed